orchestrator/index.js

```diff
@@ -0,0 +1,67 @@
+const request = require('request-promise');
+const HABANERO = require('node-red-habanero');
```

Comment:
Its a convention [used by node-red](http://nodered.org/docs/embedding).  Whenever the reference the package, they use all uppercase. we've continued that